appcenter-analytics/android/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
     compile 'com.facebook.react:react-native:+'
-    compile 'com.microsoft.appcenter:appcenter-analytics:1.8.0'
+    compile 'com.microsoft.appcenter:appcenter-analytics:1.9.0-2+56673159'
```

Comment:
Temporary version until Native SDK released, revert that after native SDK released.